src/libsyntax_pos/lib.rs

```diff
@@ -157,12 +224,7 @@ impl fmt::Debug for Span {
     }
 }
 
-pub const DUMMY_SP: Span = Span { lo: BytePos(0), hi: BytePos(0), expn_id: NO_EXPANSION };
-
-// Generic span to be used for code originating from the command line
-pub const COMMAND_LINE_SP: Span = Span { lo: BytePos(0),
-                                         hi: BytePos(0),
-                                         expn_id: COMMAND_LINE_EXPN };
```

Comment:
Same as before -- `DUMMY_SP`. The only place we used `COMMAND_LINE_SP` was used was [here](https://github.com/rust-lang/rust/pull/40597/files#diff-0d146cdcf8b61538ee2f552c43719757L84), which only happens when naming a legacy plugin crate through a debugging command line option.